src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.security.SecureRandom;
+
+public class Lottery {
+    private static final int BOUND_RANDOM = 100;
+    private final SecureRandom randomNumber = new SecureRandom();
+    private final ColorSupplier randomColor = new ColorSupplier();

```

Comment:
let's name the variable in uppercase and rename randomNumber -> RANDOM, randomColor -> COLOR_SUPPLIER